tests/test_project/test_project/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for test_project project.
+
+Generated by 'django-admin startproject' using Django 1.11.20.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '&2+vjn90r35xis@-csd(%(vl=o&xmqc!x))mv2vk=jd9^2karu'
```

Comment:
Wonder if generating it randomly here would be a better idea? Since it's for testing only.